### DISCUSSION TITLE
[@types/textfit] textFit accepts an array of HTMLElements

### DISCUSSION THREAD

**Discussion by marijn** [DefinitelyTyped/DefinitelyTyped#58703] [Issues with a @types package]
Today I noticed a discrepancy with the `textFit` API provided by `@types/textfit`. According to the projects [`README`](https://github.com/STRML/textFit#usage) the `textFit()` function accepts either a single `HTMLElement` or `HTMLElement[]`. 

Unfortunately I don't have the time to create a PR right now so this is the next best thing I could do. Let me know if I can be of assistance in any way.

[a comment by an automated account is not shown]

**Comment by mu-hun** [comment]
Thanks for report this issue. 🥰 I have created a new PR for the information you submitted. https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58733
